src/endpoints/proposals-inbox/domain/models/proposals-inbox.query-model.ts

```diff
@@ -0,0 +1,58 @@
+import {
+	Filter,
+	FindOptions,
+	MongoFindQueryParams,
+	QueryModel,
+} from '@alien-worlds/api-core';
+
+import { ProposalsInboxRequestDto } from '../../data/dtos/proposals-inbox.dto';
+
+/*imports*/
+
+/**
+ * @class
+ */
+export class ProposalsInboxQueryModel extends QueryModel {
+	/**
+	 * @returns {ProposalsInboxQueryModel}
+	 */
+	public static create(
+		model: ProposalsInboxRequestDto
+	): ProposalsInboxQueryModel {
+		const { account, dac_id, skip, limit } = model;
+		return new ProposalsInboxQueryModel(account, dac_id, skip, limit);
+	}
+
+	/**
+	 * @constructor
+	 * @private
+	 */
+	private constructor(
+		public readonly account: string,
+		public readonly dac_id: string,
+		public readonly skip: number,
+		public readonly limit: number
+	) {
+		super();
+	}
+
+	public toQueryParams(): MongoFindQueryParams<unknown> {
```

Comment:
you can use document type `WorkerProposalDocument` from @eosdac-api-common` instead of `unknown`